src/test/rgw_directory_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
What's the difference between this file and `rgw_d4n_directory_test.sh`? If there is none should one of them be removed?

Reply:
Yes, i meant to remove those files so I will do that now

Reply:
The same applies to rgw_directory_test.cc and `rgw_d4n_directory_test.cc`.